### PR TITLE
Replace DisplayBase with IGraphicsDisplay

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: Source/Meadow.Foundation.Core/Contracts/IGraphicsDisplay.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        /// <summary>
+        /// Draw a single pixel at the specified color
+        /// </summary>
+        /// <param name="x"></param>
+        /// <param name="y"></param>
+        /// <param name="color">The Meadow.Foundation color of the pixel</param>
+        public abstract void DrawPixel(int x, int y, Color color);
+
+        /// <summary>
+        /// Draw a single pixel at the specified color
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Core/Contracts/IGraphicsDisplay.cs:72]: Duplicated comment here, perhaps?

File: Source/Meadow.Foundation.Core/Displays/DisplayBase.cs [hunk 2]
```diff
@@ -1,88 +1,66 @@
...
 using Meadow.Foundation.Graphics.Buffers;
-using Meadow.Peripherals.Displays;
+using System;
 
 namespace Meadow.Foundation.Displays
 {
     /// <summary>
     /// Abstract hardware display class 
     /// </summary>
+    [Obsolete]
```
Comment (jorgedevs) [inline Source/Meadow.Foundation.Core/Displays/DisplayBase.cs:10]: Could we add a string message with a reason of why its obsolete, or encourage to use what instead?

**Review by jorgedevs — APPROVED** [review]

File: Source/Meadow.Foundation.Core/Contracts/IGraphicsDisplay.cs [hunk 3]
```diff
@@ -0,0 +1,91 @@
+﻿using Meadow.Foundation.Graphics;
```
[the remark this answers is not in the document]
Comment (adrianstevens) [reply Source/Meadow.Foundation.Core/Contracts/IGraphicsDisplay.cs]: thx - fixed!

File: Source/Meadow.Foundation.Core/Displays/DisplayBase.cs (hunk 2 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Core/Displays/DisplayBase.cs]: done!

File: Source/Meadow.Foundation.Core/Contracts/IGraphicsDisplay.cs (hunk 1 above)
Comment (adrianstevens) [reply Source/Meadow.Foundation.Core/Contracts/IGraphicsDisplay.cs]: I updated the summary for the 2nd overload

**Pull request merged**